filament/include/filament/TransformManager.h

```diff
@@ -114,7 +114,30 @@ class UTILS_PUBLIC TransformManager : public FilamentAPI {
     void setParent(Instance i, Instance newParent) noexcept;
 
     /**
-     * Set a local transform of a transform component.
+     * Returns the parent, if any, of a transform component.
```

Comment:
What happens if the entity doesn't have a parent? what's the return value?

Reply:
Renamed to numWritten.